go/oasis-node/cmd/debug/txsource/workload/governance.go

```diff
@@ -277,6 +281,170 @@ OUTER:
 	}
 }
 
+func (g *governanceWorkload) doChangeParametersProposal() error { // nolint: gocyclo
```

Comment:
Yes. If that doesn't hold for current tests, it might brake future tests or make them flaky. Not sure if there is a parameter which is safe to change. I can always keep the same parameter values, but we probably gain nothing with this.